### DISCUSSION TITLE
Set old feed page as default

### DISCUSSION THREAD

**Discussion by NitinIsFit** [community/community#27818] [Other Feature Feedback, Questions, & Ideas]
The issue is, that they broke the old homepage. It no longer shows the information it did previously! In addition, everytime I go to 'github.com' I end up on that new 'For You'-Page and thus am a click further away from the old homepage.

The 'Following'-Tab (old Homepage) is described as follows:

> The feed shows you events from people you follow and repositories you watch or star.

Since the new 'For You'-Page exists, it now only shows change to a single repository by a single user. changes by other users or in other repositories are not displayed at all.

And this is why people are so outraged! The issue isn't the new feature. The issue is, that they broke the old one.

Previously these were displayed (at least for me):

New forks and projects by Users I'm following.

New commits, releases, branches, issues, pull requests, and more for repositories I'm watching or have stared

New Releases for repos in which I only have selected 'Releases' in the watch options

New comments to issues/pull requests in which I participate.

Any comments where I'm mentioned

All Completely gone.

I now only have entries for commits by a singular user, that I'm not following, for a singular repo, that I'm following.
That users also commits ot other repos that I'm following, but those are not displayed. Also other users commit to that first repo, which also no longer show up.

Comments, Issues, branches, releases and Pull Requests in any form are entirely gone from the old 'Following'-Tab.

**PS: Even activity in my own repositories is entirely gone**

**Comment by NitinIsFit** [comment]
Originally commented by @pingger

**Comment by mpboom** [comment]
Related:
https://github.com/orgs/community/discussions/13395

**Comment by jfullstackdev** [comment]
> Comments, Issues, branches, releases and Pull Requests in any form are entirely gone from the old 'Following'-Tab.

I noticed that too, so I think the solution and valid request is to add a personal dashboard to the already existing filter, so the past will be restored
